### PR TITLE
Finalize CI pipeline for DTL Library Module

### PR DESCRIPTION
**Pull request by petehauge**
**Three changes included:**

* Ensure that all the names we use when deploying Azure resources are random.  We think we were seeing collisions that was causing test failures
* Add the pipeline status badge to the readmes for the DTL Module (setup to run nightly)
* Fix a few places on the "Should" commandlet where we didn't use the right syntax

### CODE REVIEW

**Comment by lucabol** [comment]
I will let @leovms decide to add the badge to the overall DTL Readme. We could do it later after we make sure that it is green most times :-)

File: README.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ This is a public, community-contributed repository that contains:
 
 [Az.DevTestLabs](https://github.com/Azure/azure-devtestlab/tree/master/samples/DevTestLabs/Modules/Library), for short, is a PowerShell module to simplify the management of [Azure DevTest Labs](https://azure.microsoft.com/en-us/services/devtest-lab/). It provides composable functions to create, query, update and delete labs, VMs, Custom Images and Environments. The source can be located under [/samples/DevTestLabs/Modules/Library/](https://github.com/Azure/azure-devtestlab/tree/master/samples/DevTestLabs/Modules/Library).
 
+**PowerShell Module Status:**
```
Comment (leovms) [inline README.md]: Not sure this is required here. You already have it in the library `README.md`, to which the above text already links.

Comment (petehauge) [reply README.md]: No problem - @leovms , I can get rid of the status here now.

Comment (petehauge) [reply README.md]: @leovms - done!  I got rid of this from the main readme.

**Pull request merged**